From rdunlap@xenotime.net Sun Feb  3 15:06:30 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Sun, 3 Feb 2008 15:06:25 -0800 (PST)
Subject: PCI: kernel-doc: fix pci-acpi warning
To: linux-kernel@vger.kernel.org
Cc: greg@kroah.com, len.brown@intel.com, akpm@linux-foundation.org
Message-ID: <Pine.LNX.4.64.0802031444390.3021@shark.he.net>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix PCI kernel-doc warning:
Warning(linux-2.6.24-git12//drivers/pci/pci-acpi.c:166): No description found for parameter 'hid'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/pci-acpi.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/pci/pci-acpi.c
+++ b/drivers/pci/pci-acpi.c
@@ -158,6 +158,7 @@ run_osc_out:
 /**
  * __pci_osc_support_set - register OS support to Firmware
  * @flags: OS support bits
+ * @hid: hardware ID
  *
  * Update OS support fields and doing a _OSC Query to obtain an update
  * from Firmware on supported control bits.
